src/db.py

```diff
@@ -0,0 +1,132 @@
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, Float, String, MetaData
+import sqlalchemy as sql
+import logging
+import os
+import pandas as pd
+from scipy.stats import mode
+import argparse
+
+Base = declarative_base()
+
+# set up looging config
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+DATABASE_NAME = 'instacart'
+SQLITE_DB_STRING = 'sqlite:///data/{}.db'.format(DATABASE_NAME)
+
+conn_type = 'mysql+pymysql'
+user = os.environ.get('MYSQL_USER')
+password = os.environ.get('MYSQL_PASSWORD')
+host = os.environ.get('MYSQL_HOST')
+port = os.environ.get('MYSQL_PORT')
+RDS_DB_STRING = '{}://{}:{}@{}:{}/{}'\
+                .format(conn_type, user, password, host, port, DATABASE_NAME)
+
+
+class OrderType(Base):
+    """Create a data model for order types derived from cluster centroids."""
```

Comment:
explanation not as descriptive, might help to make it clearer as to what the centroids represent and potential ranges of values

Reply:
excellent idea